setup.py

```diff
@@ -47,10 +47,10 @@ def read(fname):
             'rural_postman_star=postman_problems.examples.star.rpp_star:main'
         ]
     },
-    python_requires='>=2.7',
+    python_requires='>=3.8',
     install_requires=[
         'pandas',
-        'networkx==2.0'
+        'networkx'
```

Comment:
Looks like networkx 2.6 supports python 3.7, 3.8, 3.9.  Any reason to restrict to >=3.8 ? 

https://networkx.org/documentation/stable/release/release_2.6.html

Reply:
Also if we pin to specific python versions, perhaps also pin networkx with `>=` to the earliest version supported by your changes?  Any reason not to?

Reply:
I usually prefer to support the latest version of all package and the documentation of networkx only contains version 2.7.1 And, I'm not used to pin version but if you want to pin to 2.6 it's ok, I don't really have a good reason not to do it.